include/ring-core/base.h

```diff
@@ -89,6 +89,9 @@
 #elif defined(__MIPSEL__) && defined(__LP64__)
 #define OPENSSL_64_BIT
 #define OPENSSL_MIPS64
+#elif defined(__PPC64__) && defined(__LITTLE_ENDIAN__)
+#define OPENSSL_64_BIT
+#define OPENSSL_NO_ASM
```

Comment:
I hope that `OPENSSL_NO_ASM` isn't needed because we don't define it for any other targets.